gui/requirements.txt

```diff
@@ -6,7 +6,7 @@ matplotlib==3.8.2
 opencv-contrib-python-headless==4.6.0.66
 pandas==1.5.0
 pyside6==6.4.0.1
-python-magic==0.4.27
```

Comment:
Could we instead replace python-magic with https://pypi.org/project/puremagic/? It has the advantage of not requiring libmagic.

The code change in gui/digest.py would then be something like 

```
-        table.append([None, self.tr("MIME type"), magic.from_file(filename, mime=True)])
+        # Use the highest-confidence MIME type
+        table.append([None, self.tr("MIME type"), puremagic.magic_file(filename)[0][1]])
```